### ISSUE TITLE
find-global-symbol is acting on all clj files below project root

### ISSUE THREAD

**Issue by expez** [clojure-emacs/refactor-nrepl#49]
It should only act on files present on classpath

**Comment by expez** [comment]
```clj
(defn list-project-clj-files []
  (let [dirs-on-cp (filter fs/directory? (cp/classpath))]
    (mapcat find-clojure-sources-in-dir dirs-on-cp)))
```

I think this solves the problem, where `cp` is `clojure.java.classpath`.  But all the tests are failing with this change because the test project is permanent on classpath and the tmp project is not.  When I add the temp project to cp the original and the copy are both present and we get too many hits.

Also, not all tests clean up after themselves: we're creating extra dirs every time the tests run.

**Comment by benedekfazekas** [comment]
was not this fixed with #39?

**Comment by expez** [comment]
Indeed it was!
